testscommon/txDataBuilder/builder.go

```diff
@@ -129,6 +129,11 @@ func (builder *txDataBuilder) TransferESDT(token string, value int64) *txDataBui
 	return builder.Func(core.BuiltInFunctionESDTTransfer).Str(token).Int64(value)
 }
 
+//TrasnferNFTESDT
```

Comment:
// TransferNFTESDT - 
as it contains a typo, a missing white space and a missing -